addons/survey/views/survey_report_templates.xml

```diff
@@ -85,5 +85,19 @@
                 </t>
             </t>
         </template>
+
+        <!-- simple template with no assets, to show title on the certification preview -->
+        <template id="certification_preview">
```

Comment:
As this is not really a report qweb template, I would move it to survey_templates_print.xml file holding templates related to "printing" a survey ... makes more sense to me.